src/invidious/yt_backend/youtube_api.cr

```diff
@@ -8,6 +8,7 @@ module YoutubeAPI
   private DEFAULT_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
 
   private ANDROID_APP_VERSION = "17.29.35"
+  private ANDROID_SDK_VERSION = "29"
```

Comment:
We should use an int like the official client does.

Reply:
Yeah but I haven't been able to do it in crystal. I'm getting:
```
In src/invidious/yt_backend/youtube_api.cr:226:31

 226 | client_context["client"]["androidSdkVersion"] = client_config.android_sdk_version
                               ^
Error: no overload matches 'Hash(String, String)#[]=' with types String, Int32

Overloads are:
 - Hash(K, V)#[]=(key : K, value : V)
```
:upside_down_face: 

If you don't mind fixing it for me that would be great.

Reply:
I think this is because the value type is String... If that's some type that accepts arbitrary value (e.g. `java.lang.Object` in Java), or union type of string and number, numbers may be set as a key-value pair
(I'm not familiar with Crystal so I'm not sure what should it be)

Reply:
Yeah I know that. It's just that the Hash/object only accept String for some reason. I always have issues with the Hash in Crystal.

Reply:
...That's a tragedy